content/pt/docs/what-is-opentelemetry.md

```diff
@@ -0,0 +1,133 @@
+---
+title: O que é OpenTelemetry?
+description:
+  Uma breve explicação sobre o que é o OpenTelemetry e o que ele não é.
+weight: 150
+default_lang_commit: f17277371622b97df4c15a6cfe3ce0b22e5538ef
+---
+
+O OpenTelemetry é uma estrutura de trabalho (_framework_) e um conjunto de
+ferramentas de
+[Observabilidade](/docs/concepts/observability-primer/#what-is-observability)
+projetados para criar e gerenciar dados de telemetria como
+[rastros](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) e
+[logs](/docs/concepts/signals/logs/). Essencialmente, o OpenTelemetry é
+agnóstico em relação a fornecedores e ferramentas, o que significa que pode ser
+utilizado com uma grande variedade de sistemas (_backends_) de observabilidade,
```

Comment:
```suggestion
utilizado com uma grande variedade de backends de observabilidade,
```

Same thing here, I think (if I'm not wrong) we agreed to use `backend` as it's an well adopted term in Portuguese.